app/src/components/players/PlayerStatsTable.tsx

```diff
@@ -207,20 +207,23 @@ function PlayerSkillsTable(
 }
 
 function getSkillColumnDefinitions(player: Player, showVirtualLevels: boolean): ColumnDef<SkillValue>[] {
+  const hasSpecialEhp = (player: Player) => {
```

Comment:
this doesn't need to be a function, I think just

`const hasSpecialEHP = player.type !== PlayerType || ['f2p', 'f2p_lvl3', 'lvl3'].includes(player.build);`

would suffice

Reply:
Hey @psikoi . Of course, I will do these changes tomorrow. Apologies for the delay!